pom.xml

```diff
@@ -55,6 +55,12 @@
                 <artifactId>jsr305</artifactId>
                 <version>${version.jsr305.plugin}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/mysql/mysql-connector-java -->
+            <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>6.0.6</version>
```

Comment:
all deps versions must be defined in https://github.com/eclipse/che-dependencies/blob/master/pom.xml